samples/cpp/benchmark_app/main.cpp

```diff
@@ -699,17 +699,24 @@ int main(int argc, char* argv[]) {
         // ----------------- 8. Querying optimal runtime parameters
         // -----------------------------------------------------
         next_step();
+        slog::info << "Model: " << slog::endl;
+        auto model_name = compiledModel.get_property(ov::model_name);
+        auto optimal_nr_req = compiledModel.get_property(ov::optimal_number_of_infer_requests);
+
+        slog::info << "  " << ov::model_name << ": " << model_name << slog::endl;
+        slog::info << "  " << ov::optimal_number_of_infer_requests << ": " << optimal_nr_req << slog::endl;
+        
         // output of the actual settings that the device selected
         for (const auto& device : devices) {
-            auto supported_properties = compiledModel.get_property(ov::supported_properties);
+            auto supported_properties = core.get_property(device, ov::supported_properties);
```

Comment:
Second step in C++ now outputs all parameters from core.get_property(all_properties), like in Python

Reply:
> Second step in C++ now outputs all parameters from core.get_property(all_properties), like in Python

This change is incorrect, compiled model should be used there. Fix python tool instead of it.